conans/test/functional/editable_reference/forbidden_commands_test.py

```diff
@@ -0,0 +1,53 @@
+# coding=utf-8
+
+import textwrap
+import unittest
+
+from conans.model.ref import ConanFileReference
+from conans.paths.package_layouts.package_editable_layout import CONAN_PACKAGE_LAYOUT_FILE
+from conans.test.utils.tools import TestClient, TestServer
+
+
+class ForbiddenCommandsTest(unittest.TestCase):
+    conanfile = textwrap.dedent("""\
+        from conans import ConanFile
+
+        class APck(ConanFile):
+            pass
+        """)
+
+    def setUp(self):
+        self.reference = ConanFileReference.loads('lib/version@user/name')
+
+        test_server = TestServer()
+        self.servers = {"default": test_server}
+        self.t = TestClient(servers=self.servers, users={"default": [("lasote", "mypass")]})
+        self.t.save(files={'conanfile.py': self.conanfile,
+                           CONAN_PACKAGE_LAYOUT_FILE: "", })
+        self.t.run('export  . {}'.format(self.reference))
+        self.t.run('install --editable=. {}'.format(self.reference))
+        self.assertTrue(self.t.client_cache.installed_as_editable(self.reference))
+
+    def test_export(self):
+        self.t.run('export . {}'.format(self.reference), assert_error=True)
+        self.assertIn("Operation not allowed on a package installed as editable", self.t.out)
+
+    def test_create(self):
+        self.t.run('create . {}'.format(self.reference), assert_error=True)
+        self.assertIn("Operation not allowed on a package installed as editable", self.t.out)
+
+    def test_remove(self):
+        self.t.run('remove -f {}'.format(self.reference), assert_error=True)
+        self.assertIn("Operation not allowed on a package installed as editable", self.t.out)
```

Comment:
I think the remove should be allowed, of course, to remove the local cache folder and not the project.